src/components/shared/components/Authorization/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { firebaseConnect, isLoaded, isEmpty } from 'react-redux-firebase';
+
+
+const Authorization = (Component) => {
+    const BaseAuth = ({ auth, ...props }) => {
+        console.log(auth);
+        if (isLoaded(auth) && !isEmpty(auth)) {
+            return <Component auth={auth} {...props} />
+        } else if (isLoaded(auth) && isEmpty(auth)) {
```

Comment:
De else if zou niet nodig hoeven zijn, enkel redirecten naar login (dus zonder de return null en return '')

Reply:
m nee dat gaat niet.
de als de auth nog moet worden geladen, dan redirect ie meteen. dus ik moet iig wachten tot die geladen is.